packages/boim/config/commonConfig.js

```diff
@@ -0,0 +1,134 @@
+const path = require("path");
+
+const TerserPlugin = require("terser-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+
+const root = path.resolve("./");
+const client = path.resolve(root, "../../");
+const isDev = process.env.NODE_ENV === "development";
+
+function createConfig(entries, outputOption, additionalAlias, mod, emit, plugins) {
+  return {
+    target: "node",
+    mode: isDev ? "development" : "production",
+    entry: { ...entries },
+    output: {
+      filename: outputOption.filename ? outputOption.filename : "[name]",
+      path: outputOption.path,
+      library: outputOption.libraryName,
+      libraryTarget: "umd",
+      globalObject: "this",
+      assetModuleFilename: "../public/[name][ext]",
+    },
+    resolve: {
+      extensions: [".js", ".jsx", ".json", ".ts", ".tsx"],
+      alias: additionalAlias ? {
+        react: `${client}/node_modules/react`,
+        "react-dom": `${client}/node_modules/react-dom`,
+        ...additionalAlias,
+      } : {
+        react: `${client}/node_modules/react`,
+        "react-dom": `${client}/node_modules/react-dom`,
+      },
```

Comment:
혹시 additionalAlias가 없을 경우 아래처럼 코드를 작성하면 에러가 발생하여 삼항연산자로 작성해두신 걸까요?
```suggestion
      alias: {
        react: `${client}/node_modules/react`,
        "react-dom": `${client}/node_modules/react-dom`,
        ...additionalAlias,
      } ,
```